src/test/java/com/ncc/aif/ExamplesAndValidationTest.java

```diff
@@ -1397,7 +1390,7 @@ void valid() {
                 // setup() already makes type assertions on the entity, event,
                 // and relation objects, justified by a text justification.
 
-                // Note that if you use makeRelationInEventForm, you will need to use the Jena API to obtain
+                // Note that if you use the deprecated makeRelationInEventForm, you will need to use the Jena API to obtain
```

Comment:
Should we even mention this? Will this encourage users to look at makeRelationInEventForm instead of ignoring it (as they should)?

Reply:
Yeah, I was wondering that myself.  Removed.